reader/signals.py

```diff
@@ -80,18 +78,70 @@ def pre_save_chapter(sender, instance, **kwargs):
             ) + timedelta(days=7)
             instance.series.save()
 
+@receiver(post_init, sender=Chapter)
+def remember_original_series_of_chapter(sender, instance, **kwargs):
+    instance.old_chapter_number = str(instance.slug_chapter_number()) if instance.chapter_number is not None else None
+    instance.old_series_slug = str(instance.series.slug) if hasattr(instance, 'series') else None
+    instance.old_group_id = str(instance.group.id) if hasattr(instance, 'group') else None
+
 
 @receiver(post_save, sender=Chapter)
 def post_save_chapter(sender, instance, **kwargs):
+    # If the group or series or the chapter number has changed, move all chapter file
```

Comment:
Nice, this was critically missing considering these three attributes made up the absolute path of the media content.